catalog/app/containers/Bucket/Summarize.tsx

```diff
@@ -83,13 +84,11 @@ const FileThemeContext = React.createContext(FileThemes.Overview)
 
 const useSectionStyles = M.makeStyles((t) => ({
   root: {
+    marginTop: t.spacing(2),
```

Comment:
does this look right on xs screens? bc it was specifically used on sm+ screens where we have rounded boxes and disabled on xs screens where we have straight boxes which collapse and are separated just by a thin divider line

Reply:
It doesn't look rignt on small screen, because there rounded border and no margin between them.

But intention is now clear, I'll fix this in another PR, and revert this change here

Reply:
Border radius on previous block, files l
![Screenshot from 2022-04-01 10-23-32](https://user-images.githubusercontent.com/533229/161215763-0f0f0b0d-ad80-4b3c-bb24-3a83b9c07a84.png)
isting

Reply:
Fixed here https://github.com/quiltdata/quilt/pull/2773